beacon-chain/state/stategen/replay.go

```diff
@@ -81,7 +81,7 @@ func (_ *State) replayBlocks(
 func (s *State) loadBlocks(ctx context.Context, startSlot, endSlot types.Slot, endBlockRoot [32]byte) ([]interfaces.SignedBeaconBlock, error) {
 	// Nothing to load for invalid range.
 	if endSlot < startSlot {
-		return nil, fmt.Errorf("start slot %d >= end slot %d", startSlot, endSlot)
+		return nil, fmt.Errorf("start slot %d > end slot %d", startSlot, endSlot)
```

Comment:
I think this was correct. `>=` is the inverse of `<`

Reply:
startSlot should <= endSlot, but the error is `startSlot > endSlot`, right?

Reply:
Perhaps we can flip the if statement on 83 to match the original error message.

```go
	if startSlot >= endSlot {
		return nil, fmt.Errorf("start slot %d >= end slot %d", startSlot, endSlot)
	}
```

Reply:
`SetStartSlot()` and `SetEndSlot()` are both inclusive, so if `startSlot == endSlot`, we still could get one block to load. 

The only place to call this function is 
```golang
blks, err := s.LoadBlocks(ctx, startState.Slot()+1, targetSlot, bytesutil.ToBytes32(summary.Root))
```
in `beacon-chain/state/stategen/getter.go`

if `startState.Slot() + 1 == targetSlot`, there still one block of slot to load.

I wonder if it is OK that `startSlot == endSlot`, so the following statement would be better?

```golang
	if startSlot > endSlot {
		return nil, fmt.Errorf("start slot %d > end slot %d", startSlot, endSlot)
	}
```

Reply:
OK i understand now and I agree with your assessment. My suggestion `>=` is not correct.

Your suggestion looks good to me. It's a pure flip without any changes. Thanks!